modules/dev-tools/src/ts-smoosh/README.md

```diff
@@ -0,0 +1,116 @@
+## TS Smoosh
```

Comment:
Move to docs?

Reply:
Some of the implementation details like running tests aren't relevant outside the context of this source. Added a new doc `ocular-tsify.md` alongside other commands that contains a subset that is relevant to the end user